### PR TITLE
Remove events listeners done + some style added

### PR DESCRIPTION
**Pull request by LauraMihalca**

### CODE REVIEW

File: project/assets/scripts/component/component-constructor.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
 const ComponentConstructor = function (options) {
-    if (options.login) {
+    if (options === 'login') {
```
Comment (softvision-alexcarcea) [inline project/assets/scripts/component/component-constructor.js:2]: In cases where you're doing repetitive checks against the same value, you can use `switch` / `case` constructs for cleaner code. Optional, for your consideration.

File: project/assets/scripts/component/component-constructor.js [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
         return {
             view: new LoginPage(),
-            setEvents: new LoginEvents(),
+            events: new LoginEvents(),
```
Comment (softvision-alexcarcea) [inline project/assets/scripts/component/component-constructor.js:5]: Would we be able to hide event details from the outside world? Could we encapsulate the event setup inside a module initialization routine?

File: project/assets/scripts/controller/add-page/add-page.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// (function () {
```
Comment (softvision-alexcarcea) [inline project/assets/scripts/controller/add-page/add-page.js:1]: If no longer needed, let's go ahead and remove.

**Review by softvision-alexcarcea — APPROVED** [review]

File: project/assets/scripts/component/component-constructor.js (hunk 1 above)
Comment (LauraMihalca) [reply project/assets/scripts/component/component-constructor.js:2]: I'll use switch, thanks!

File: project/assets/scripts/controller/add-page/add-page.js (hunk 3 above)
Comment (LauraMihalca) [reply project/assets/scripts/controller/add-page/add-page.js:1]: 👍

**Pull request merged**